include/zephyr/bluetooth/gatt.h

```diff
@@ -712,7 +712,7 @@ typedef uint8_t (*bt_gatt_attr_func_t)(const struct bt_gatt_attr *attr,
  *
  *  Iterate attributes in the given range matching given UUID and/or data.
  *
- *  @param start_handle Start handle.
+ *  @param start_handle Starting attribute handle.
  *  @param end_handle End handle. Often set to start_handle + attr_count or
```

Comment:
```suggestion
 *  @param start_handle Starting attribute handle.
 *  @param end_handle Ending attribute handle. Often set to start_handle + attr_count or
```
More cohesive to change both start and end.